vars/preview.yml

```diff
@@ -5,17 +5,20 @@ maintenance_mode: live
 api:
   routes:
     - dm-api-{env}.apps.internal
+    - dm-api-{env}.cloudapps.digital/_metrics
   memory: 2GB
   egress_to_applications:
     - search-api
 
 search-api:
   routes:
     - dm-search-api-{env}.apps.internal
+    - dm-api-{env}.cloudapps.digital/_metrics
```

Comment:
Should this and the AV API one below be `dm-search-api-{env}.cloudapps.digital/_metrics`

Reply:
Good catch 👍

Reply:
Done 👍